TRB_label_assist/models.py

```diff
@@ -23,7 +23,14 @@
 import emission.analysis.modelling.tour_model_first_only.evaluation_pipeline as ep
```

Comment:
you should remove these imports as well. There is really no reason for us to keep using `tour_model_first_only`. Is is not used in the production server code; it has been replaced by `trip_model` there.

Reply:
done.